pkg/vm/cli/cli.go

```diff
@@ -128,6 +128,30 @@ Example:
 > step 10`,
 		Func: handleStep,
 	},
+	{
+		Name: "stepInto",
```

Comment:
These are to be entered by the user into the CLI prompt. I don't think mixedCase will make users happy.

Reply:
hm hm could be so=)
stepinto then?

Reply:
Yep.